pkg/cli/image/manifest/manifest.go

```diff
@@ -541,15 +501,19 @@ func ManifestsFromList(ctx context.Context, srcDigest digest.Digest, srcManifest
 	}
 }
 
-// TODO: Remove support for v2 schema in 4.9
+// PutManifestInCompatibleSchema just calls ManifestService.Put right now.
+// No schema conversion is happening anymore. Instead of using this function,
+// call ManifestService.Put directly.
+//
+// Deprecated
 func PutManifestInCompatibleSchema(
 	ctx context.Context,
 	srcManifest distribution.Manifest,
 	tag string,
 	toManifests distribution.ManifestService,
 	ref reference.Named,
-	blobs distribution.BlobService, // support schema2 -> schema1 downconversion
-	configJSON []byte, // optional, if not passed blobs will be used
+	blobs distribution.BlobService, // unused
```

Comment:
Did you check that this function is being used somewhere else (other than within the oc code base)?. Maybe we can simply drop these unused flags?.

Reply:
Not within the openshift org, so I guess it can be removed.